Classes/WPEditorView.m

```diff
@@ -275,8 +275,17 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
             [newValue getValue:&newSize];
         
             if (newSize.height != self.lastEditorHeight) {
-                [self refreshVisibleViewportAndContentSize];
+                
+                // First make sure that the content size is not changed without us recalculating it.
+                //
+                self.webView.scrollView.contentSize = CGSizeMake(self.frame.size.width, self.lastEditorHeight);
```

Comment:
@aerych - **Regarding your first comment:** I've actually given up on properly calculating the height here, and decided to do it after a very small delay.  The code that sets the last calculated height basically makes sure that the web view doesn't change the content size on its own (which is causing some janky animations when switching the focus from the title to the content).

This code is definitely not of my liking, but I think the kind of changes we need to remove this are kind of too drastic this close to the editor's 4.8 release.  My idea is to bring up that there are some fragile spots in the editor and that there are some changes we may need to make to fix those spots.

**Regarding your second comment:** sounds definitely good to me.  It's just one of those things I find hard to remember at times, so it's also a very nice thing that you brought it up in the review.  I definitely encourage it becoming a standard / convention.